frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -29,8 +53,8 @@ async function pollFunnel(params = {}) {
     return result
 }
 
-const cleanFunnelParams = (filters) => {
-    return {
+const cleanFunnelParams = (filters: Partial<FilterType>): Partial<FilterType> => {
+    const params = {
         ...filters,
         ...(filters.date_from ? { date_from: filters.date_from } : {}),
```

Comment:
I realize this code was already present, but this much spreading is hard to read. It might be acceptable to just spread `filters`, or if the goal is to coerce falsy values to `undefined` (which is happening here), perhaps spread something like `Object.fromEntries(Object.entries(filters).filter(([_key, value]) => Boolean(value)))`

Reply:
I'm confused why we're coercing falsy values to undefined here 😮  (for my own understanding)

Reply:
Isn't undefined a falsy value too 😢